plugins/main/bridge/bridge_suite_test.go

```diff
@@ -15,13 +15,26 @@
 package main
 
 import (
-	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
 	"testing"
 )
 
+var (
+	resolvConf string
+	err        error
+)
+
 func TestBridge(t *testing.T) {
 	RegisterFailHandler(Fail)
+
+	resolvConf, err = newResolvConf()
```

Comment:
Would it be more clean to defer the cleanup from within the test to eliminate the need for global variables?

Reply:
It's not uncommon to have global vars in the test_suite used by the test files.
And I think it's more of a good practice to have the cleanup on the test_suite (as mentioned [here](https://onsi.github.io/ginkgo/#suite-setup-and-cleanup-beforesuite-and-aftersuite))
Plus, having a `defer deleteResolvConf(resolvConf)` within the test i.e in the Describe node won't work.

Reply:
@austinvazquez please let me know if this is ok with you, or do you have a different idea for how to make it cleaner?